src/mlpack/tests/string_utility_test.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file string_utility_test.cpp
+ * @author Ryan Curtin
+ *
+ * Tests for String Utility Class.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/core/data/bow_encoding.hpp>
+#include "mlpack/core/boost_backport/boost_backport_string_view.hpp"
+#include <mlpack/core/data/tokenizer/char_split.hpp>
+#include <boost/test/unit_test.hpp>
+#include "test_tools.hpp"
+
+using namespace mlpack;
+using namespace mlpack::data;
+using namespace std;
+
+BOOST_AUTO_TEST_SUITE(StringUtilityTest);
+
+boost::string_view tokenizer(boost::string_view& str)
```

Comment:
Cool, I think we can also just pass this as a lambda function?

Reply:
This function is already there in other PR, so when it will be rebased , it woudn't make much difference, to reduce line, I just used same function here.

Still if you want, we can go with lamda function. I have used lamda function there, in other PR.

Reply:
Fine with me.